test/test_exporter.py

```diff
@@ -204,103 +220,51 @@ def test_dynatrace_metadata_enrichment_with_default_attributes(
-
-        # the second export misses the metric with delta=3 because the name was invalid
-        second_expected = "my.instr,l1=v1,l2=v2,dt.metrics.source=opentelemetry count,delta=2 {0}\n" \
-            .format(str(self._test_timestamp_millis))
-
-        exporter = DynatraceMetricsExporter()
-        result = exporter.export(metrics)
-
-        self.assertEqual(MetricExportResult.SUCCESS, result)
-
-        mock_post.assert_any_call(
-            self._ingest_endpoint,
-            data=second_expected,
-            headers=self._headers)
-
-        mock_post.assert_any_call(
-            self._ingest_endpoint,
-            data=first_expected,
-            headers=self._headers)
-
-    @patch.object(requests.Session, 'post')
-    @patch('dynatrace.metric.utils.dynatrace_metrics_api_constants'
-           '.DynatraceMetricsApiConstants.payload_lines_limit')
-    def test_invalid_metricname_skipped(self, mock_const, mock_post):
-        mock_post.return_value = self._get_session_response()
-        mock_const.return_value = 2
-
-        metrics = []
-        for n in range(4):
-            data_point = self._create_sum_data_point(n)
-            if n == 3:
-                # for the last metric, we create one with an invalid name
-                metric = Metric(attributes=self._attributes,
-                                name="",
-                                point=data_point,
-                                description="",
-                                unit="1",
-                                resource=Resource({}),
-                                instrumentation_scope=InstrumentationScope(
-                                    name="dynatrace.opentelemetry.metrics.export",
-                                    version="0.0.1"))
-                metrics.append(metric)
-            else:
-                metrics.append(self._create_record(data_point))
-
-        first_expected = "my.instr,l1=v1,l2=v2,dt.metrics.source=opentelemetry count,delta=0 {0}\n" \
-                         "my.instr,l1=v1,l2=v2,dt.metrics.source=opentelemetry count,delta=1 {0}\n" \
-            .format(self._test_timestamp_millis)
-
-        # the second export misses the metric with delta=3 because the name was invalid
-        second_expected = "my.instr,l1=v1,l2=v2,dt.metrics.source=opentelemetry count,delta=2 {0}\n" \
-            .format(self._test_timestamp_millis)
-
-        exporter = DynatraceMetricsExporter()
-        result = exporter.export(metrics)
-
-        self.assertEqual(MetricExportResult.SUCCESS, result)
-
-        mock_post.assert_any_call(
-            self._ingest_endpoint,
-            data=second_expected,
-            headers=self._headers)
-
-        mock_post.assert_any_call(
-            self._ingest_endpoint,
-            data=first_expected,
-            headers=self._headers)
+    @parameterized.expand([
+        ("",),
+        (".",)
+    ])
+    def test_invalid_metricname_skipped(self, instrument_name):
```

Comment:
Does this test test that one metric is dropped, but also that the chunking works? Is there any reason to have the chunking test here as well?